source/_includes/custom/navigation.html

```diff
@@ -9,7 +9,7 @@
       </a>
       <div class="nav-collapse collapse">
         <ul class="nav">
-          <li {% if page.url == '/index.html' %}class="selected"{% endif %}>
+          <li {% if page.url == '/index.html' or page.url contains '/blog/'%}class="selected"{% endif %}>
```

Comment:
@jfewins I have updated the condition to handle the archive page! it should work now https://github.com/sujaysudheenda/cerner.github.io/commit/a7abd62d6a1967e1d1669212389e92547390b222


sorry for keeping this PR alive for so long :)